fusor-ember-cli/app/styles/custom.scss

```diff
@@ -795,3 +795,7 @@ h1.num-ose-node {
     display: block;
   }
 }
+
+.navbar-outer {
+  z-index: 1200 !important;
```

Comment:
This overwrites this, but not sure how it works with `z-index: 1`
https://github.com/theforeman/foreman/blob/develop/app/assets/stylesheets/topbar.scss#L84

when in bootstrap, the `.navbar-fixed-top` menu, which is the second menu has
`z-index: 1030`

so, I created a `z-index` higher than 1030